### PR TITLE
Declarations for better config builder usage

### PR DESCRIPTION
**Pull request by enrico-padovani**

### CODE REVIEW

File: src/config-builder.js [hunk 1]
```diff
@@ -374,11 +374,6 @@ export class KendoConfigBuilder {
     return this;
   }
 
-  kendoSlider(): KendoConfigBuilder {
```
Comment (JeroenVinke) [inline src/config-builder.js]: why was this removed?

**Comment by JeroenVinke** [comment]
Thanks!

File: src/config-builder.js (hunk 1 above)
Comment (enrico-padovani) [inline src/config-builder.js]: Because it was duplicated

Comment (JeroenVinke) [inline src/config-builder.js]: I see, all good then 😄

**Comment by JeroenVinke** [comment]
I'll try and get a release out either today or tomorrow

**Comment by enrico-padovani** [comment]
Good, nice to contribute somehow to this awesome project!
